docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md

```diff
@@ -7,6 +7,12 @@ in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.16.0] - 2023-11-17
+
+### Added
+
+- Added `recurrence rule` option to `downloadCalendar`, this can be used to set recurring event in the calendar. 
```

Comment:
```suggestion
- Added `recurrenceRule` option to `downloadCalendar`, this can be used to set recurring events in the calendar. Rule syntax can be found at https://datatracker.ietf.org/doc/html/rfc5545#section-3.3.10.
```